src/components/navigation/breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '../../base/icons/Icon';
+
+function Breadcrumb({ breadcrumbClasses, children, ...props }) {
+  return (
+    <div className={breadcrumbClasses}>

```

Comment:
To build more on @Darrken's [comment](https://github.com/WTW-IM/es-components/pull/97/files#r155866010), having `breadcrumbClasses` instead of `className` isn't very intuitive. I'm thinking we can simplify this by simply spreading `props` on the top level.

```js
function Breadcrumb({ children, ...props }) {
  return (
    <div {...props}>
      { children mapping }
    </div>
  );
}
```